ccm_web/server/src/api/api.interfaces.ts

```diff
@@ -10,3 +10,8 @@ export interface Globals {
     roles: string[]
   }
 }
+
+export interface Course {
+  id: number
+  name: string
+}
```

Comment:
This is just a minimal version of the Canvas Course entity: https://canvas.instructure.com/doc/api/courses.html

I could maybe see moving this to the `canvas` module.

Reply:
I did end up doing this. I don't really know if it was correct, but I'm trying to keep very-Canvas specific stuff in that module.